app/controllers/rapidfire/attempts_controller.rb

```diff
@@ -1,6 +1,6 @@
 module Rapidfire
   class AttemptsController < Rapidfire::ApplicationController
-    if Rails::VERSION::MAJOR == 5
+    if Rails::VERSION::MAJOR >= 5
       before_action :find_survey!
     else
       before_filter :find_survey!
```

Comment:
@aksh93 We can remove before_filter completely with rails 4.2 and above. https://edgeguides.rubyonrails.org/4_2_release_notes.html#action-pack-notable-changes

Reply:
Alright, let's do that.